src/cheri_regs.sail

```diff
@@ -195,23 +179,7 @@ function ext_rvfi_init () = {
   x12 = root_cap_mem;
   x13 = root_cap_mem;
   x14 = root_cap_mem;
-  x15 = root_cap_mem;
-  x16 = root_cap_mem;
-  x17 = root_cap_mem;
-  x18 = root_cap_mem;
-  x19 = root_cap_mem;
-  x20 = root_cap_mem;
-  x21 = root_cap_mem;
-  x22 = root_cap_mem;
-  x23 = root_cap_mem;
-  x24 = root_cap_mem;
-  x25 = root_cap_mem;
-  x26 = root_cap_mem;
-  x27 = root_cap_mem;
-  x28 = root_cap_mem;
-  x29 = root_cap_mem;
-  x30 = root_cap_mem;
-  x31 = root_cap_mem;
+  x15 = root_cap_mem
```

Comment:
Did you remove the semicolon intentionally?! Makes no difference, just curious.

Reply:
Oh, apparently.  I guess I thought `; }` was malformed.  I can put it back to shrink the diff a line.